docs/dev-tools/console/console.asciidoc

```diff
@@ -1,5 +1,9 @@
 [[console-kibana]]
-== Run API requests
+== Run API requests with Console
+
+Console lets you interact with {es} and {kib} APIs from within {kib}.
```

Comment:
🚗 Say we decide to remove all feature documentation duplication from serverless docs, we might want to remove all mention of Kibana in favor of something like "the UI" which works equally well whether you're working on hosted or serverless.

Reply:
This is worth debating. Do we want to not say "kibana", or do we want to embrace it and just plainly establish that Kibana is where all solutions, regardless of hosting/architecture, run. "The UI" is vague since we unfortunately have several UIs for now. And telling a consistent story across hosting options also has some benefits.